docs/ui-coverage/configuration/attributefilters.mdx

```diff
@@ -2,186 +2,13 @@
 title: 'Attribute Filters | Cypress UI Coverage'
 description: 'The `attributeFilters` configuration property allows users to specify patterns for attributes and their values that should not be used for identifying and grouping elements.'
 sidebar_label: attributeFilters
+sidebar_position: 60
 ---
 
-<ProductHeading product="ui-coverage" />
+<ProductHeading product="ui-coverage" /># attributeFilters
```

Comment:
also broken 
![Screenshot 2025-06-02 at 11 38 53 AM](https://github.com/user-attachments/assets/ec292054-a678-4d69-8a74-845657cc9f32)

Reply:
Thank you - no idea how these got committed that way, 🙃